jest/setup.js

```diff
@@ -52,7 +55,9 @@ jest
         // Ensure this doesn't throw.
         try {
           Object.keys(dataBlob).forEach(key => {
-            this.items += dataBlob[key] && dataBlob[key].length;
+            const isImmutableMap = Map.isMap(dataBlob[key]);
+            this.items += isImmutableMap ?
+              (dataBlob[key] && !dataBlob[key].isEmpty()) : (dataBlob[key] && dataBlob[key].length);
```

Comment:
`this.items` should be numeric, don't you need to get the length here if it's immutable?